scripts/license-header.txt

```diff
@@ -0,0 +1,17 @@
+{%- if attrs.git_file_modified_year != props["inceptionYear"] -%}
+Copyright © {{ props["inceptionYear"] }}-{{ attrs.git_file_modified_year }} {{ props["copyrightOwner"] }}. All rights reserved.
+{%- else -%}
+Copyright © {{ props["inceptionYear"] }} {{ props["copyrightOwner"] }}. All rights reserved.
+{%- endif %}
```

Comment:
The copyright range should actually be from the file's creation date, rather than the project's creation date.

I played around with this, and this seems to work:

```suggestion
Copyright ©{{ " " }}{%- if attrs.git_file_modified_year != attrs.git_file_created_year -%}{{ attrs.git_file_created_year }}-{{ attrs.git_file_modified_year }}{%- else -%}{{ attrs.git_file_created_year}}{%- endif -%}{{ " " }}{{ props["copyrightOwner"]}}. All rights reserved.
```